{    "Logs": "Added object temp to s3 bucket lambda-cpp-me-south-1
    	clock precision level: 3
    	Cache line size: 64 B
    	Found an address that falls on two cache lines: 0x55555759cffc
    	Baseline sample: Size- 1483, Mean- 6952
    	[Lambda-  4] Phase, Position, Bit, Sent, Read, Lat Size, Lat Mean, Lat Std, Lat Max, Lat Min, Base Mean, Base Lat, PValue
    	[Lambda-  4]   0         7    0     0     0      1435      7069     5344    41940     5108       6952     5448 1.000314010093848
    	[Lambda-  4]   0         6    0     0     0      1500      7249     5666    42222     5092       6952     5448 1.002449697391525
    	[Lambda-  4]   0         5    0     0     0      1500      7291     6259    61498     5128       6952     5448 1.002952560958907
    	[Lambda-  4]   0         4    0     0     0      1500      8714    20253   458526     5116       6952     5448 1.023032604116880
    	[Lambda-  4]   0         3    0     0     0      1500      8656    21939   516168     5090       6952     5448 1.018821462665665
    	[Lambda-  4]   0         2    1     1     1      1500         0        0        0        0       6952     5448 1.018821462665665
    	[Lambda-  4]   0         1    0     0     0      1472     28857   787378 30207764     5082       6952     5448 1.002233342663051
    	[Lambda-  4]   0         0    0     0     0      1449     10253    29859   551864     5134       6952     5448 1.027702730458844
    	[Lambda-4] Phase 0, Id read: 4",    "Boot ID": "0a4d574a-c8cb-4fdb-9c4a-44e720b53ec0",    "MAC Address": "36:ED:F3:6C:FF:24",    "Phase 1": "4",    "Phases": "1",    "Start Time": "2020-08-14 00:25:55",    "End Time": "2020-08-14 00:26:24",    "Id": "4",    "Request ID": "9dbc75a0-6902-4b58-afda-aa3055d6f96e",    "Success": "1",    "Error": ""}